barretenberg/cpp/src/barretenberg/relations/databus_lookup_relation.hpp

```diff
@@ -174,6 +186,11 @@ template <typename FF_> class DatabusLookupRelationImpl {
         const auto read_tag_m =
             CoefficientAccumulator(BusData<bus_idx, AllEntities>::read_tags(in)); // does row contain data being read
         const Accumulator read_tag(read_tag_m);
+        // Relation checking: is_read_gate == 1 || read_tag == 1
+        // Important note: the relation written below assumes that is_read_gate and read_tag are boolean values,
+        // if not, fixing one of the two, the return value is a linear function in the other variable and can be
+        // set to an arbitrary value independent of the fixed value. Check the boolean_check subrelation for more
```

Comment:
```suggestion
        // set to an arbitrary value independent of the fixed value. See the boolean_check subrelation for more
```
Might want to explicitly say something like "and this is guaranteed by the boolean_check subrealtion", otherwise its not clear if this comment is raising concern some unhandled issue or something the user needs to worry about